lib/rspec_controller_helpers/resourceful.rb

```diff
@@ -1,23 +1,43 @@
 RSpec.shared_examples_for 'a resourceful controller' do |model, raw_options|
-  let(:options) { raw_options || {} }
+  let(:options) { ({ endpoint_formats: {} }).merge(raw_options) }
```

Comment:
`endpoint_formats` may be a bit long. Given its a controller tester, `formats` should do.